backend/package.json

```diff
@@ -20,7 +20,7 @@
   ],
   "engineStrict": true,
   "engines": {
-    "node": "^18.14"
+    "node": "v18.18.0"
```

Comment:
Does this still allow 18.18.1 for example or is it locked at exactly 18.18.0?

Reply:
it's locked at the lts/Hydrogen version of Node - I think that's what we want for this ticket, right @ShawnTurple ?